taskcluster/scripts/toolchain/compile_qt_6.ps1

```diff
@@ -74,6 +83,75 @@ Set-Location $FETCHES_PATH/qt-everywhere-src-$QT_VERSION
   -qt-zlib  `
   -openssl-runtime `
   -prefix $BUILD_PREFIX `
+} else {
+  # For newer qt versions, let's trim what we dont need.
```

Comment:
A comment that explains where these options are documented would be helpful to the reader. I found some in qtbase\config_help.txt.

Reply:
That's great! Added more context on how i drilled to that list